ext/zip/php_zip.stub.php

```diff
@@ -66,23 +66,18 @@ function zip_entry_compressionmethod($zip_entry): string|false {}
 
 class ZipArchive implements Countable
 {
-    /** @var int|null */
-    public $lastId;
-
-    /** @var int|null */
-    public $status;
-
-    /** @var int|null */
-    public $statusSys;
-
-    /** @var int|null */
-    public $numFiles;
-
-    /** @var string|null */
-    public $filename;
-
-    /** @var string|null */
-    public $comment;
+    /** @readonly */
+    public int $lastId = -1;
```

Comment:
I think it makes sense to declare default values, but please correct me if I'm wrong.

Reply:
I don't think default values make sense for dynamically produced properties.

Reply:
I mainly thought about documentation purposes, but I'l remove these in the next commit